ee/desktop/user/server/server_darwin.go

```diff
@@ -68,12 +81,21 @@ func (s *UserServer) getSecureEnclaveKey(w http.ResponseWriter, r *http.Request)
 		// apple site where you can search for error codes, just enter error code
 		// https://developer.apple.com/bugreporter/
 		if strings.Contains(err.Error(), "-25300") {
+			s.slogger.Log(r.Context(), slog.LevelInfo,
+				"secure enclave key does not exist",
+				"err", err,
+			)
 			http.Error(w, "key not found", http.StatusNotFound)
 			return
 		}
 
+		s.slogger.Log(r.Context(), slog.LevelError,
+			"encountered unexpected error, cannot determine if key exists in secure enclave",
+			"err", err,
+		)
+
 		// encountered some other error, cannot confirm if key exists
-		http.Error(w, fmt.Errorf("encounter error, cannot determine if key exists in secure enclave: %w", err).Error(), http.StatusInternalServerError)
+		http.Error(w, fmt.Errorf("encounter unexpected error, cannot determine if key exists in secure enclave: %w", err).Error(), http.StatusInternalServerError)
```

Comment:
same question as above, if these are errors we don't really control I kinda wonder if appropriate to put them directly in the response